mscore/uploadscoredialog.cpp

```diff
@@ -69,6 +69,7 @@ UploadScoreDialog::UploadScoreDialog(LoginManager* loginManager)
                            .arg("</a>"));
       QFont font = licenseHelp->font();
       font.setPointSize(8);
+      font.setItalic(true);
       licenseHelp->setFont(font);
 
```

Comment:
Yes, now it is, and all "comments" or help messages are italic. Look a bit better.